File: src/app/gkill/dao/user_config/device_struct.go
package user_config

type DeviceStruct struct {
	ID string `json:"id"`

	UserID string `json:"user_id"`

	DeviceName string `json:"device_name"`

	Device string `json:"device"`

	ParentFolderID string `json:"parent_folder_id"`

	Seq int `json:"seq"`

	CheckWhenInited bool `json:"check_when_inited"`

	IsDir bool `json:"is_dir"`

	IsOpenDefault bool `json:"is_open_default"`
}

File: src/classes/datas/config/device-struct.ts
'use strict'

export class DeviceStruct {

    id: string

    user_id: string

    device: string

    device_name: string

    parent_folder_id: string | null

    seq: Number

    check_when_inited: boolean

    is_dir: boolean

    is_open_default: boolean

    constructor() {
        this.id = ""
        this.user_id = ""
        this.device = ""
        this.device_name = ""
        this.parent_folder_id = ""
        this.seq = 0
        this.check_when_inited = false
        this.is_dir = false
        this.is_open_default = false
    }

}


